src/SFA.DAS.ApprenticeCommitments.Web/Pages/Apprenticeships/Confirm.cshtml.cs

```diff
@@ -13,33 +14,61 @@ namespace SFA.DAS.ApprenticeCommitments.Web.Pages.Apprenticeships
     [Authorize]
     public class ConfirmApprenticeshipModel : PageModel
     {
-        private readonly AuthenticatedUserClient _client;
+        //private readonly AuthenticatedUserClient _client;
+
+        private readonly IOuterApiClient _client;
+        private readonly AuthenticatedUser _authenticatedUser;
 
         [BindProperty(SupportsGet = true)]
-        public HashedId? ApprenticeshipId { get; set; }
+        public HashedId ApprenticeshipId { get; set; }
 
         public bool? EmployerConfirmation { get; set; } = null;
         public bool? TrainingProviderConfirmation { get; set; } = null;
         public bool? ApprenticeshipDetailsConfirmation { get; set; } = null;
         public bool? RolesAndResponsibilitiesConfirmation { get; set; } = null;
         public bool? HowApprenticeshipWillBeDeliveredConfirmation { get; set; } = null;
 
-        public ConfirmApprenticeshipModel(AuthenticatedUserClient client)
+        public string Forwardlink => $"/apprenticeships/{ApprenticeshipId.Hashed}/transactioncomplete";
+
+        public bool AllConfirmed
+        {
+            get
+            {
+                return EmployerConfirmation.HasValue && (bool)EmployerConfirmation == true
+                    && TrainingProviderConfirmation.HasValue && (bool)TrainingProviderConfirmation == true
+                    && ApprenticeshipDetailsConfirmation.HasValue && (bool)ApprenticeshipDetailsConfirmation == true
+                    && RolesAndResponsibilitiesConfirmation.HasValue && (bool)RolesAndResponsibilitiesConfirmation == true;
+            }
+        }
+
+        public ConfirmApprenticeshipModel(IOuterApiClient client, AuthenticatedUser authenticatedUser)
         {
             _client = client;
+            _authenticatedUser = authenticatedUser;
         }
 
         public async Task OnGetAsync()
         {
             _ = ApprenticeshipId ?? throw new ArgumentNullException(nameof(ApprenticeshipId));
 
-            var apprenticeship = await _client.GetApprenticeship(ApprenticeshipId.Id);
-            
+            //var apprenticeship = await _client.GetApprenticeship(ApprenticeshipId.Id);
```

Comment:
Can we remove this comment or will it be needed later?

Reply:
Same as above regarding AuthenticatedUserClient